lamp/files/etc/update-motd.d/99-one-click

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+#
+# Configured as part of the DigitalOcean 1-Click Image build process
+
+myip=$(hostname -I | awk '{print$1}')
+cat <<EOF
+********************************************************************************
+
+Welcome to DigitalOcean's One-Click LAMP Droplet.
```

Comment:
Should we be using `1-Click` everywhere?

Reply:
Yes